Gems/Atom/Feature/Common/Code/Source/DiffuseGlobalIllumination/DiffuseProbeGridPreparePass.cpp

```diff
@@ -29,10 +30,11 @@ namespace AZ
         DiffuseProbeGridPreparePass::DiffuseProbeGridPreparePass(const RPI::PassDescriptor& descriptor)
             : RPI::RenderPass(descriptor)
         {
-            if (!AZ_TRAIT_DIFFUSE_GI_PASSES_SUPPORTED)
+            if (!(RHI::RHISystemInterface::Get()->GetDevice()->GetFeatures().m_rayTracing && AZ_TRAIT_DIFFUSE_GI_PASSES_SUPPORTED))
```

Comment:
Please remove the m_rayTracing check from this class, it is executed even on non-RT platforms.

Reply:
If you check the commits themselves, there is a second temp commit that can be easily reverted when we add RT support. It was disabled mainly for testing and avoiding asserts on load.

Reply:
I intentionally made those changes in a designated commit so it would he easy to revert